lib/geoblacklight/dataingest.rb

```diff
@@ -16,14 +16,12 @@ class DataIngest
     "dc:creator": {required: true},
```

Comment:
creator, description, format, language, publisher,, ispartof,  spatial, layer:geom_type and layer_id optional?

Reply:
Although these fields are not required in the geoblacklight solr schema, these are required fields for GeoData.